config/intellij-idea-inspections.properties

```diff
@@ -8,5 +8,6 @@ idea.exclude.patterns=.idea/**;\
               src/site/resources/js/google-analytics.js;\
               src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/ParseTreeBuilder.java;\
               src/test/java/com/puppycrawl/tools/checkstyle/grammar/javadoc/ParseTreeBuilder.java;\
+              src/test/java/com/puppycrawl/tools/checkstyle/internal/ArchUnitTest.java;\
```

Comment:
it is not ok.

Reply:
@romani I have removed and added `@noinspection`. Thanks a lot for the guidance.